native/src/components/HeaderContainer.ts

```diff
@@ -47,6 +47,18 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
           route: routeName as typeof simpleRoutes extends (infer U)[] ? U : never
         })
       : null
+  const newsRouteShareUrl =
```

Comment:
Will refactor this in a separate PR but works fine for now (https://issues.tuerantuer.org/browse/IGAPP-635)

Reply:
See https://github.com/Integreat/integreat-app/pull/524